src/yoroi-wallets/cardano/mocks/index.ts

```diff
@@ -0,0 +1,815 @@
+            name: '546f6b656e3131',
+          },
+          {
+            amount: '2418889379',
+            assetId: 'c85f714f2187021c7bab53741f659d0c5b1a6e7529d32b7794ff051c.474f4c44',
+            policyId: 'c85f714f2187021c7bab53741f659d0c5b1a6e7529d32b7794ff051c',
+            name: '474f4c44',
+          },
+          {
+            amount: '2463889377',
+            assetId: 'c868cdb63090661d815bac251aad5fcffaef94cf099e6cd81df33490.474f4c44',
+            policyId: 'c868cdb63090661d815bac251aad5fcffaef94cf099e6cd81df33490',
+            name: '474f4c44',
+          },
+          {
+            amount: '100',
+            assetId: 'ce3c3f372d4b277c3a583421bda2799a62b5b5105076b03d1e28b07b.53544b443130',
+            policyId: 'ce3c3f372d4b277c3a583421bda2799a62b5b5105076b03d1e28b07b',
+            name: '53544b443130',
+          },
+          {
+            amount: '30499999987788',
+            assetId: 'd27197682d71905c087c5c3b61b10e6d746db0b9bef351014d75bb26.6e69636f696e',
+            policyId: 'd27197682d71905c087c5c3b61b10e6d746db0b9bef351014d75bb26',
+            name: '6e69636f696e',
+          },
+          {
+            amount: '25867',
+            assetId:
+              'e4214b7cce62ac6fbba385d164df48e157eae5863521b4b67ca71d86.438bb31d1920ad7fff1e0e93cab8a887eaa0b0c6754f578631f13389c3cdb0cd',
+            policyId: 'e4214b7cce62ac6fbba385d164df48e157eae5863521b4b67ca71d86',
+            name: '438bb31d1920ad7fff1e0e93cab8a887eaa0b0c6754f578631f13389c3cdb0cd',
+          },
+          {
+            amount: '416592',
+            assetId:
+              'e4214b7cce62ac6fbba385d164df48e157eae5863521b4b67ca71d86.60a585ee984a47140f7c201f238d48f89585d1a9f42687750626db3a906b050a',
+            policyId: 'e4214b7cce62ac6fbba385d164df48e157eae5863521b4b67ca71d86',
+            name: '60a585ee984a47140f7c201f238d48f89585d1a9f42687750626db3a906b050a',
+          },
+          {
+            amount: '8613',
+            assetId:
+              'e4214b7cce62ac6fbba385d164df48e157eae5863521b4b67ca71d86.86e90c911f058c3ebeb95a120eedd311caff3bb49d5b29ff8a9bad42005b041f',
+            policyId: 'e4214b7cce62ac6fbba385d164df48e157eae5863521b4b67ca71d86',
+            name: '86e90c911f058c3ebeb95a120eedd311caff3bb49d5b29ff8a9bad42005b041f',
+          },
+          {
+            amount: '9',
+            assetId: 'ecd07b4ef62f37a68d145de8efd60c53d288dd5ffc641215120cc3db.',
+            policyId: 'ecd07b4ef62f37a68d145de8efd60c53d288dd5ffc641215120cc3db',
+            name: '',
+          },
+        ],
+      },
+    ],
+    lastUpdatedAt: '2022-06-12T23:46:47.000Z',
+    submittedAt: '2022-06-12T23:46:47.000Z',
+    blockNum: 3626415,
+    blockHash: '3be6615c2711f8c85e5777d1a060682bba507551d99d3568569fe8dfb7dcc690',
+    txOrdinal: 5,
+    epoch: 210,
+    slot: 357991,
+    withdrawals: [],
+    certificates: [],
+    validContract: true,
+    scriptSize: 0,
+    collateralInputs: [],
+    memo: null,
+  },
+}
+
+export const mockApi = {
```

Comment:
delete mockApi, 
create the mock in the test

Reply:
It is not posible to use out-of-scope variables inside the mock factory function so I would need to add everything from there in the factory function.

Reply:
A the mocked history is a big one mock